### PR TITLE
Add note about Scala Native configuration to getting started guide

### PR DESCRIPTION
**Pull request by rorygraves**

### CODE REVIEW

File: docs/gettingstarted.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ In addition to being compiled into JVM bytecode, Rsc is compiled with Scala
 Native. If you'd like to compile native binaries or run native tests, check out
 Scala Native documentation for
 [additional instructions](http://www.scala-native.org/en/latest/user/setup.html).
+When installing Scala Native install the optional `bdw-gc` and `re2` packages, as they
+are requried by the build.
```
Comment (xeno-by) [inline docs/gettingstarted.md]: Looks like a typo: `s/requried/required/`.

Comment (rorygraves) [reply docs/gettingstarted.md]: Whoops - fixing now

**Comment by rorygraves** [comment]
@xeno-by Updated as per review

**Comment by xeno-by** [comment]
Thanks for the contribution! Please sign the CLA (https://cla-assistant.io/twitter/reasonable-scala?pullRequest=18), and we're good to go.

**Comment by rorygraves** [comment]
You are welcome - CLA signed

**Comment by xeno-by** [comment]
@rorygraves Your name is temporarily missing in CONTRIBUTORS.md, but we're working on it. Most likely, we'll delete CONTRIBUTORS.md and let https://github.com/twitter/reasonable-scala/graphs/contributors take care of that.

**Comment by ShaneDelmore** [comment]
Contributors.md is removed.  One less thing to manually keep up to date.

**Pull request merged**